embedded-aerospike-enterprise/pom.xml

```diff
@@ -14,7 +14,7 @@
     <artifactId>embedded-aerospike-enterprise</artifactId>
 
     <properties>
-        <aerospike-client.version>7.2.0</aerospike-client.version>
+        <aerospike-client.version>8.0.1</aerospike-client.version>
```

Comment:
Is Aerospike client 8.x compatible with Aerospike 6.x? Cannot find it here: https://aerospike.com/developer/client/client_matrix

Reply:
Need waiting update client matrix version dependency from Aerospike team.